subsys/trusted_storage/Kconfig

```diff
@@ -0,0 +1,195 @@
+# Trusted storage subsystem configuration options
```

Comment:
We've put both trusted storage and protected storage in this Kconfig, so I think it is better named as "secure storage", which is the term used to describe both of them elsewhere as well.

Reply:
So having it called secure storage subsystem?
Not sure if I understood you correctly.

Reply:
Disscued offline